siriuspy/siriuspy/clientconfigdb/types/global_config.py

```diff
@@ -6,6 +6,8 @@
 from siriuspy.pwrsupply.csdev import DEFAULT_WFM_OTHERS as \
     _DEFAULT_WFM_OTHERS
 
+from siriuspy.clientconfigdb.types.as_diagnostics import _bpms
```

Comment:
Change here to a local import.

Reply:
It didn't work:

```
In [1]: import siriuspy.clientconfigdb.types.global_config as gc
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[1], line 1
----> 1 import siriuspy.clientconfigdb.types.global_config as gc

File ~/projects/scripts/src/siriuspy/siriuspy/siriuspy/clientconfigdb/__init__.py:3
      1 """Clientconfig subpackage."""
----> 3 from .configdb_client import ConfigDBClient, ConfigDBException
      4 from .configdb_document import ConfigDBDocument
      5 from .pvsconfig import PVsConfig

File ~/projects/scripts/src/siriuspy/siriuspy/siriuspy/clientconfigdb/configdb_client.py:14
     10 import numpy as _np
     12 from .. import envars as _envars
---> 14 from . import _templates
     17 class ConfigDBClient:
     18     """Perform operation on configuration database."""

File ~/projects/scripts/src/siriuspy/siriuspy/siriuspy/clientconfigdb/_templates.py:7
      4 import copy as _copy
      5 import numpy as _np
----> 7 from . import types as _types
      9 # NOTE: values for key string labels ending with char '*' have not their
     10 #       sizes compared with a reference if their lists or tuples!
     11 
     12 # NOTE: cannot use tuple values in configuration dictionaries since conversion
     13 #       through json looses track of tuple|list type.
     15 _CONFIG_TYPES_DICT = None

File ~/projects/scripts/src/siriuspy/siriuspy/siriuspy/clientconfigdb/types/__init__.py:9
      7 for loader, module_name, is_pkg in _pkgutil.walk_packages(__path__):
      8     CONFIG_TYPES.append(module_name)
----> 9     module = loader.find_module(module_name).load_module(module_name)
     10     exec('%s = module' % module_name)

File ~/projects/scripts/src/siriuspy/siriuspy/siriuspy/clientconfigdb/types/global_config.py:9
      4 from siriuspy.pwrsupply.csdev import DEFAULT_WFM_FBP as \
      5     _DEFAULT_WFM_FBP
      6 from siriuspy.pwrsupply.csdev import DEFAULT_WFM_OTHERS as \
      7     _DEFAULT_WFM_OTHERS
----> 9 from .as_diagnostics import _bpms
     12 _OFF = 0
     13 _ON = 1

ImportError: attempted relative import with no known parent package
```

I talked to @anacso17, and she said the issue seems to be due to how `types/__init__.py` works.